papaparse.js

```diff
@@ -51,8 +54,19 @@ if (!Array.isArray)
 		return {};
 	})();
 
+
+	function getWorkerBlob() {
+		try {
```

Comment:
Why adding a try/catch? IIUC return null will make the code break on next line so it seems better to fail here directly in order to get a better stack trace

Reply:
Yeah, I was on the fence about this. In practice it doesn't throw unless on older browsers, and in my practice I've nulled the value to indicate fallback behavior... but I can see the value in just blowing up here. Will update.